src/player.js

```diff
@@ -2549,6 +2552,7 @@ export default class Player extends FakeEventTarget {
     let currentOrConfiguredAudioLang = this._playbackAttributesState.audioLanguage || playbackConfig.audioLanguage;
     this._setDefaultTrack<TextTrack>(this._getTextTracks(), currentOrConfiguredTextLang, offTextTrack);
     this._setDefaultTrack<AudioTrack>(this._getAudioTracks(), currentOrConfiguredAudioLang, activeTracks.audio);
+    this._setDefaultVideoTrack();
```

Comment:
the issue is talking about playlist but this will also now work for non playlist cases - is this intended?
I think we should put this behind a flag so it can be controlled, and the flag should be generic like `preserveVideoSelection` and by default it will be `false` and when loading playlist, internally in kaltura-player we will set it to true.
this will help us avoid breaking changes as well.
I also think we should think of preserving ABR vs manual bitrates as a feature so we can also use it for ABR, so maybe define `preserveVideoSelection` as an enum with options:
* `NONE`(as we have today) 
* `MANUAL`(the current feature in this PR) 
* `ABR`(the new proposed one)
* `ALL`(both manual and ABR)

Reply:
as discussed with Ron L this is for manual selection only. and for any change media flow (not playlist only). it's very similar to captions selection or audio selection that we keep while the session